.ci/docker/requirements-ci.txt

```
@@ -101,9 +101,11 @@ numba==0.55.2 ; python_version == "3.10"
 #test that import: test_numba_integration.py
 #For numba issue see https://github.com/pytorch/pytorch/issues/51511
 
-#numpy
+# This need to match with numba version above
+numpy==1.20.3 ; python_version <= "3.9"
+numpy==1.21.2 ; python_version == "3.10"
```

I don't like current rule as it excludes python 3.11 and 3.12, so I wonder if something like that would be more inclusive
```suggestion
numpy<=1.21.2 
```